.github/workflows/ci.yml

```diff
@@ -84,7 +84,11 @@ jobs:
         with:
             lfs: true
       - name: Install stable Rust toolchain
-        uses: dtolnay/rust-toolchain@stable
+        uses: dtolnay/rust-toolchain@master
+        with:
+          # FIXME: Use rust-toolchain@stable and remove this when 1.91 is stable
```

Comment:
This will still be necessary, though, until that target is tier 2 or build-std is stable, right?